ccloud-observability/start.sh

```diff
@@ -64,13 +64,11 @@ echo "$OUTPUT" | jq .
 export METRICS_API_KEY=$(echo "$OUTPUT" | jq -r ".key")
 export METRICS_API_SECRET=$(echo "$OUTPUT" | jq -r ".secret")
 export CLOUD_CLUSTER=$CLUSTER
-rm .env 2>/dev/null
+export CLOUD_CONNECTORS=$(confluent connect list -o json | jq -c '. | map(.id)')
```

Comment:
1. What happens if this is empty?
2. What happens if there is more than one connector?

Reply:
1. Will always be true for the demo as-run, as there's no connectors.  The resulting `.env` contains:

```
CLOUD_CONNECTORS=[]
```

The resulting `prometheus.yml` contains:

```
      "resource.connector.id": []
```

This doesn't appear to bother our scrape query build.

2. Works if hand-edited to contain an array e.g. `["lcc-xxxxx", "lcc-yyyyy"], which is what the `jq` also generates, should you run this against a cluster with connectors.